### PR TITLE
add binary sysfs node to show ERT CQ for easy of debug

### PR DESCRIPTION
**Pull request by chienwei-lan**

### CODE REVIEW

**Comment by dayeh-xilinx** [comment]
retest this please

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/command_queue.c [hunk 1]
```diff
@@ -545,6 +595,12 @@ static int command_queue_probe(struct platform_device *pdev)
 		}
 	}
 
+	err = sysfs_create_group(&pdev->dev.kobj, &cmd_queue_attr_group);
+	if (err) {
+		xocl_err(&pdev->dev, "create ert_cq sysfs attrs failed: %d", err);
```
Comment (mamin506) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/command_queue.c:600]: later on, we might have more and more sysfs node for other purpose. Should we print "Create command queue sysfs node group failed" here?

**Review by mamin506 — CHANGES_REQUESTED** [review]
Thanks Chien-wei. This is a good to have for debug hang issue.
Just a minor change.

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/command_queue.c (hunk 1 above)
Comment (mamin506) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/command_queue.c:600]: ert_cq will be the sub-dev name. Not just the argument.

**Review by mamin506 — APPROVED** [review]

**Pull request merged**